api2-war/src/main/java/eu/europeana/api2/v2/utils/ApiConstants.java

```diff
@@ -8,7 +8,10 @@ public final class ApiConstants {
   public static final String API_KEY_SERVICE_CLIENT_DETAILS = "apiKeyClientDetails";
   public static final String X_API_KEY = "X-Api-Key";
 
+  public static final String TOKEN_ENDPOINT_PROPERTY = "token_endpoint";
```

Comment:
This is redundant. As these constants are already present in `AuthenticationConfig` class. Please use that